Tests/XCUITests/HomePageSettingsUITest.swift

```diff
@@ -40,8 +40,6 @@ class HomePageSettingsUITests: BaseTestCase {
         navigator.goto(HomeSettings)
         XCTAssertTrue(app.tables.cells["Firefox Home"].exists)
         XCTAssertTrue(app.tables.cells["HomeAsCustomURL"].exists)
-        XCTAssert(app.tables.cells["WallpaperSettings"].exists)
```

Comment:
There's no "Wallpaper Settings" and "Recommended by Pocket" from the settings any more.
<img width="144" alt="Screenshot 2022-11-21 at 5 31 47 PM" src="https://user-images.githubusercontent.com/1740517/203171316-0d4d1c37-dab2-4620-be90-31c527c3559c.png">

Reply:
I see.. as we talked... let's add the options that are available now. Recommended by Pocket is still there as a switch...
![Simulator Screen Shot - iPhone 14 - 2022-11-22 at 16 24 31](https://user-images.githubusercontent.com/1897507/203353016-49c1c2c5-2b49-4f47-8a94-9c742d041338.png)